modules/Layout/Boilerplate/Boilerplate.module.scss

```diff
@@ -19,19 +20,6 @@
     color: $color-base-white;
 }
 
-.aboutUs {
```

Comment:
This makes the layout a bit different from the current display, when there is no social information.

Current:
<img width="1219" alt="Screen Shot 2022-08-19 at 12 41 00" src="https://user-images.githubusercontent.com/1641218/185580538-eb3813aa-cca8-48aa-ade7-838f1c9f064f.png">

This PR:
<img width="1251" alt="Screen Shot 2022-08-19 at 12 41 07" src="https://user-images.githubusercontent.com/1641218/185580555-99b47df7-4184-41ec-a00e-4aeaebed5eb1.png">

Personally, I think this looks a bit better, but curious what @iamhugofelix thinks.

Reply:
The only reason why the about column (in the footer) is full-width, regards covering the empty space left from the lack of the contact column. 
Having it occupying 2/3 of the page and leaving that much "white space" on the right of the page, makes the footer look broken, especially because other elements (such as the contact card and the subscription form) occupy the entire width of the page.

Taking into consideration that 4/5 Prezly Newsrooms include at least 1 contact (which makes the contact columns to be visible), I don't see a reason to change the layout on it, other than cleaning the code (but that I'll need to leave to Oleg to evaluate).